fixed namespace in otel-demo example (#244)
kyma-project/examples: trace-demo/README.md

   open http://localhost:8080
   ````

> 2. To verify that traces arrive in the Jaeger backend, assuming you deployed a [Jaeger in-cluster](./../jaeger/) using the same namespace as used for the demo app:

Reviewer: ```suggestion
2. Verify that traces arrive in the Jaeger backend. If you deployed [Jaeger in your cluster](./../jaeger/) using the same Namespace as used for the demo app, run:
```

Can't edit line 78 but there, I suggest "1. Verify the frontend:" without the "To" (because step 3 and 4 don't have "To..." either)
